File: sdkwork-ai-sdk-csharp/Models/ContextResponse.cs
using System;
using System.Collections.Generic;
using System.Text.Json.Serialization;

namespace Ai.Models
{
    public class ContextResponse
    {
        public string? Id { get; set; }
        public string? Object { get; set; }
        public int? Created { get; set; }
        public string? Model { get; set; }
        public string? Mode { get; set; }
        public int? Ttl { get; set; }
        public TruncationStrategy? TruncationStrategy { get; set; }
        public Usage? Usage { get; set; }
    }
}

File: sdkwork-ai-sdk-go/types/context_response.go
package types

// 上下文缓存响应
type ContextResponse struct {
	Id string `json:"id"`
	Object string `json:"object"`
	Created int `json:"created"`
	Model string `json:"model"`
	Mode string `json:"mode"`
	Ttl int `json:"ttl"`
	TruncationStrategy TruncationStrategy `json:"truncationStrategy"`
	Usage Usage `json:"usage"`
}
